os/fs/smartfs/smartfs_smart.c

```diff
@@ -693,6 +714,18 @@ static int smartfs_sync_internal(struct smartfs_mountpt_s *fs, struct smartfs_of
 	}
 #endif							/* CONFIG_SMARTFS_USE_SECTOR_BUFFER */
 
+#ifdef CONFIG_SMARTFS_USE_SECTOR_BUFFER
```

Comment:
I think these codes can be optimized with codes in line 665

Reply:
Sorry could you explain please?

Reply:
Ohkay you mean adding this to the part with
`#ifdef CONFIG_SMARTFS_USE_SECTOR_BUFFER`
Because otherwise this part of code is not used.